catalyst_voices/test/common/bip39/seed_phrase_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:bip39/bip39.dart' as bip39;
+import 'package:catalyst_voices/common/bip39/seed_phrase.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group(SeedPhrase, () {
```

Comment:
Please add a test that generates some seed phrase and assert that is has exactly 12 words (I think this is the seed phrase length we agreed for)